osc/commandline.py

```diff
@@ -2312,17 +2314,27 @@ def do_request(self, subcmd, opts, *args):
             ## Workaround: fetch all, and filter on client side.
 
             ## FIXME: date filtering should become implemented on server side
+
             for result in results:
-                if days == 0 or result.state.when > since or result.state.name == 'new':
-                    if (opts.interactive or conf.config['request_show_interactive']) and not opts.non_interactive:
-                        ignore_reviews = subcmd != 'review'
-                        request_interactive_review(apiurl, result, group=opts.group,
-                                                   ignore_reviews=ignore_reviews,
-                                                   source_buildstatus=source_buildstatus)
+                filtered = 0
+                for action in result.actions:
+                    if action.type == 'group' or not opts.target_package_filter:
+                        continue
+                    filter_pattern = re.compile(opts.target_package_filter)
```

Comment:
As already discussed in IRC it matched the right submitrequests. The group is not really needed. I will remove the grouping from the example to avoid confusion.